Makefile

```diff
@@ -51,6 +51,10 @@ RSRCS = \
 
 #%}
 
+#%}
+
+#%}
+
```

Comment:
?

Reply:
Oh dear, that looks like a very useful change ;-) I still use Pe with it´s Makefile mode to add source files to the Makefile. It likes to add these extra lines when doing that. In this case I added and then removed a source file again, leaving only these lines changed in the commit. I´m going to clean it up and edit the Makefile by hand next time.

Reply:
I´ve cleaned up the lines in question